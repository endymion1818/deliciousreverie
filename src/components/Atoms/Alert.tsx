import React from 'react'
import styled from 'styled-components'
import { borderradius, size } from "../tokens";

const Alert = styled.div`
    background: rgba(255,255,255,0.2);
    border-radius: ${borderradius.small};
    padding: ${size.half} ${size.double};
`

export default Alert
