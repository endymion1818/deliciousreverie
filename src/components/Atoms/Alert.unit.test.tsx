import { cleanup, render } from '@testing-library/react'
import React from 'react'
import Alert from './Alert'

describe('<Alert />', () => {
  beforeEach(cleanup)
  it('Should render', () => {
    const { container } = render(<Alert />)
    expect(container).toMatchSnapshot()
  })
})
