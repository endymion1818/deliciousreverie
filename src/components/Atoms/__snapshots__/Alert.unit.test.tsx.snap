// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Alert /> Should render 1`] = `
.c0 {
  background: rgba(255,255,255,0.2);
  border-radius: 0.25rem;
  padding: 0.5rem 2rem;
}

<div>
  <div
    class="c0"
  />
</div>
`;
